cloudbuild.yaml

```diff
@@ -0,0 +1,10 @@
+steps:
+  - name: 'gcr.io/cloud-builders/docker'
+    args:
+      - 'build'
+      - '--tag=gcr.io/$PROJECT_ID/apps/samson_secret_puller:$SHORT_SHA'
```

Comment:
Using git tags is more descriptive.

Reply:
yes, but then we need fuzzy tags too (when commit is not tagged) which will get ugly
also most other projects use revision, so consistency is nice too :)